src/main/webapp/pipe.js

```diff
@@ -1,4 +1,4 @@
-function updatePipelines(divNames, errorDiv, view, fullscreen, showChanges, timeout) {
+function updatePipelines(divNames, errorDiv, view, fullscreen, showChanges, aggregatedChangesGroupingPattern, timeout) {
```

Comment:
maybe later we should replace this params with one single "config" object with a set of params.